### ISSUE TITLE
Global tweaks

### ISSUE THREAD

**Issue by DevTimur** [MoonManModding/EmergingTechnology#11]
Here is a list of tweaks I would like to see.
1) Bed should accept only allowed blocks, and only 1 instance of it, so i couldn't insert (for example)  iron ingots or whole stack of dirt. Only 1 piece of dirt shift+clicked per bed. Maybe even make so when you rightclick bed with dirt/sand/gravel/etc. it would automatically put 1 item in inventory. Setting up big field kinda frustrating.
2) Lights should share power similar to beds sharing water. Bulbs sould behave same way as dirt with bed. 1 item per inventory, or even rightclicking with bulb automatically put 1 bulb in.
3) "The One Probe" mod shows there is power storage in beds, what it used for?
4) You should add some tooltips to blocks describing what they do. For example, at what range does lamp work? If i place it like 8 blocks above sugarcane, will it work? How about it working with trees? How far can i place lamp from a sapling vertically, but it still have effect?
5) Optimize your blocks so they cause less chunk updates. For example if i have like 100+ beds it cause severe fps drop when ever i'm runing out of water. You shouldn't update bed block depending on water level. Make it eathier have water, or don't have.
6) Make soil block bigger, size almost as big as bed itself. Small pile of dirt laying in pool of water look silly.

That's all for now

**Comment by MoonManModding** [comment]
Thanks for the suggestions and feedback.

1. & 2. are planned, but I had to implement the custom bulbs/media features first.

In regards to 3, there is a config option that requires beds to be powered before transferring water to other adjacent beds.

4. All the machine GUIs are being redone for 1.2.0 which will include detailed information about the effect of bulbs, media and fluids on the current crop.

5. Which version of the mod are you using? The beds shouldn't be causing chunk updates when transferring water.

6. Tried this but it just looks like a big plant pot. The design will change once I've found one I'm happy with 😄
